src/app/portlets/dot-edit-page/main/dot-edit-page-main/dot-edit-page-main.component.ts

```diff
@@ -1,19 +1,35 @@
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
 import { Observable } from 'rxjs/Observable';
 import { DotRenderedPageState } from '../../shared/models/dot-rendered-page-state.model';
+import { Subscription } from 'rxjs/Subscription';
+import { filter } from 'rxjs/operators';
 
 @Component({
     selector: 'dot-edit-page-main',
     templateUrl: './dot-edit-page-main.component.html',
     styleUrls: ['./dot-edit-page-main.component.scss']
 })
-export class DotEditPageMainComponent implements OnInit {
+export class DotEditPageMainComponent implements OnInit, OnDestroy {
     pageState: Observable<DotRenderedPageState>;
+    private routeEventsSub: Subscription;
 
-    constructor(private route: ActivatedRoute) {}
+    constructor(private route: ActivatedRoute, private router: Router) {}
 
-    ngOnInit() {
+    ngOnInit(): void {
+        this.loadData();
+        this.routeEventsSub = this.router.events
+            .pipe(filter((anyEvent: any) => event instanceof NavigationEnd && event.url.indexOf('/edit-page/content') > -1))
```

Comment:
I dont undestand what exactly is happend here:

"event instanceof NavigationEnd && event.url.indexOf('/edit-page/content')"

this code is not self explain so I think is better extract it into a method